### PR TITLE
refactor: [ANDROAPP-6170] Remove old form

### PR DESCRIPTION
**Pull request by andresmr**
Remove old xml form

### CODE REVIEW

File: app/src/androidTest/java/org/dhis2/usescases/teidashboard/TeiDashboardTest.kt [hunk 1]
```diff
@@ -240,7 +239,7 @@ class TeiDashboardTest : BaseTest() {
 
         eventRobot(composeTestRule) {
             waitToDebounce(600)
-            fillRadioButtonForm(4)
+//            fillRadioButtonForm(4)
```
Comment (DavidAparicioAlbaAsenjo) [inline app/src/androidTest/java/org/dhis2/usescases/teidashboard/TeiDashboardTest.kt:242]: can you remove comments?

File: app/src/androidTest/java/org/dhis2/usescases/event/EventTest.kt [hunk 2]
```diff
@@ -105,7 +103,7 @@ class EventTest : BaseTest() {
         }
 
         eventRobot(composeTestRule) {
-            fillRadioButtonForm(radioFormLength)
+//            fillRadioButtonForm(radioFormLength)
```
Comment (DavidAparicioAlbaAsenjo) [inline app/src/androidTest/java/org/dhis2/usescases/event/EventTest.kt:106]: remove comments

**Review by DavidAparicioAlbaAsenjo — APPROVED** [review]

File: app/src/androidTest/java/org/dhis2/usescases/enrollment/EnrollmentTest.kt [hunk 3]
```diff
@@ -12,19 +13,20 @@ import org.junit.runner.RunWith
...
+    @Ignore("Not working in new form, fixed on ANDROAPP-6179")
     @Test
     fun shouldShowDateEditionWarningMessage() {
         setupCredentials()
         setDatePicker()
         prepareTeiWithAutogeneratedEvents(ruleEnrollment)
 
         enrollmentFormRobot {
-            clickOnDateOfBirth()
+//            clickOnDateOfBirth()
```
Comment (xavimolloy) [inline app/src/androidTest/java/org/dhis2/usescases/enrollment/EnrollmentTest.kt:29]: Could you please remove commented code

File: app/src/androidTest/java/org/dhis2/usescases/flow/syncFlow/SyncFlowTest.kt [hunk 4]
```diff
@@ -83,7 +82,7 @@ class SyncFlowTest : BaseTest() {
         }
 
         eventRobot(composeTestRule) {
-            fillRadioButtonForm(4)
+//            fillRadioButtonForm(4)
```
Comment (xavimolloy) [inline app/src/androidTest/java/org/dhis2/usescases/flow/syncFlow/SyncFlowTest.kt:85]: could you please remove commented code if it's no longer necessary?

File: app/src/androidTest/java/org/dhis2/usescases/event/EventTest.kt (hunk 2 above)
Comment (andresmr) [reply app/src/androidTest/java/org/dhis2/usescases/event/EventTest.kt:106]: This is a commented test, but the method does not exists anymore so I left the method commented to have context on the steps when fixing it.

**Review by xavimolloy — APPROVED** [review]

**Pull request merged**